repairnator/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/inspectors/TestJobStatus.java

```diff
@@ -70,12 +78,12 @@ public void testGatheringPatches() throws IOException {
         RepairnatorConfig.getInstance().setRepairTools(new HashSet<>(Arrays.asList(nopolRepair.getRepairToolName(), npeRepair.getRepairToolName())));
 
         cloneStep.addNextStep(new CheckoutBuggyBuild(inspector, true))
-                .addNextStep(new TestProject(inspector))
-                .addNextStep(new GatherTestInformation(inspector, true, new BuildShouldFail(), false))
-                .addNextStep(new ComputeClasspath(inspector, true))
-                .addNextStep(new ComputeSourceDir(inspector, true, false))
-                .addNextStep(nopolRepair)
-                .addNextStep(npeRepair);
+            .addNextStep(new TestProject(inspector))
```

Comment:
is that only a formatting change?

Reply:
Yes, one tab character gets removed in each line.

Reply:
it's good practice to avoid formatting only changes

Reply:
Ok, I will follow it.